rero_ils/modules/acquisition/acq_orders/views.py

```diff
@@ -75,7 +75,8 @@ def order_notification_preview(order_pid):
               f'"{language}" language'
         current_app.logger.error(msg)
         response['message'] = [{'type': 'error', 'content': msg}]
-        tmpl_file = 'rero_ils/vendor_order_mail/eng.tpl.txt'
+        language = current_app.config.get('RERO_DEFAULT_LANGUAGE', 'eng')
```

Comment:
Why not using: BABEL_DEFAULT_LANGUAGE

Reply:
I think because it is two characters ('en') and templates prefixes have 3 ('eng')...